packages/field-base/test/field-mixin.test.js

```diff
@@ -878,10 +878,9 @@ const runTests = (defineHelper, baseMixin) => {
         helper = element.querySelector('[slot=helper]');
       });
 
-      describe('aria-labelledby', () => {
```

Comment:
It seems that all the "group field" tests become now unnecessary because the mixin treats them equally with non group fields. I would propose removing them.

Reply:
Yeah, makes sense. At first I thought about letting it stay here to make it explicity about group and non-group fields behavior, but we can FFS here.